python/cudf/cudf/tests/test_array_ufunc.py

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2020-2022, NVIDIA CORPORATION.
+# Copyright (c) 2020-2023, NVIDIA CORPORATION.
```

Comment:
Why did this change?

Reply:
This file was edited in https://github.com/rapidsai/cudf/pull/13083/ but looks like the style checker missed adding it locally and has done now.